src/CTA.WebForms2Blazor/Helpers/ControlHelpers/UniversalDirectiveAttributeMap.cs

```diff
@@ -14,29 +15,34 @@ public static class UniversalDirectiveAttributeMap
         public const string CodeBehind = "CodeBehind";
         public const string ClassNameAttr = "ClassName";
         public const string LanguageAttr = "Language";
+        public const string SourceAttr = "Src";
 
         // Decided to use a universal attribute map because many of the attributes seem to be shared between
         // directives and using attribute allow lists by directive seemed like a better solution
-        public static IDictionary<string, Func<string, IEnumerable<DirectiveMigrationResult>>> AttributeMap
+        // Func takes in a string array to allow for variable number of string inputs,
+        // which may be required for different attributes
+        public static IDictionary<string, Func<string[], IEnumerable<DirectiveMigrationResult>>> AttributeMap
```

Comment:
Same thing as Jon previously mentioned with static dictionary, we should use ConcurrentDictionary here as well.